routes/adminapi.js

```diff
@@ -61,23 +62,26 @@ router.post('/saveword', function(req, res) {
     if (!(config.COOKIE_NAME in req.cookies))
       return res.status(400).send('Cookie not provided.');
     var sessionId = req.cookies[config.COOKIE_NAME];
-    // if sessionId is not whitelisted, throw error
+    util.getProfileFromGoogle(sessionId,
```

Comment:
bhai session-id is not firebase token so this function cant be used
dont we store email id in db?